client/src/pages/events/join-event.tsx

```diff
@@ -14,6 +14,17 @@ const JoinButton = styled(Button)`
   }
 `;
 
+const JoinedSuccessfully = styled.p`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 10px;
+  border-radius: 2px;
+  text-align: center;
+  font-weight: normal;
+  color: #00A52E;
+  background: #E4FFE3;
+`;
```

Comment:
this panel is out of app styles.

proper success message is in confirm-email-page